File: Oxiyopal.Models/ViewModels/HistorialDeFactura.cs
using Oxiyopal.Models.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace Oxiyopal.Models.ViewModels
{
    public class HistorialDeFactura : EntityBase<int>
    {
        public int CilindroId { get; set; }
        public Cilindro Cilindro { get; set; }

        public int FacturaId { get; set; }
        public Factura Factura { get; set; }
    }
}

File: Oxiyopal.WebUI/ClientApp/src/app/models/ViewModels/HistorialDeFactura.ts
import { Cilindro } from '../Cilindro';
import { Factura } from '../Factura';
import { EntityBase } from '../Base/EntityBase';

export interface HistorialDeFactura extends EntityBase<number> {
    cilindroId: number;
    cilindro: Cilindro;

    facturaId: number;
    factura: Factura;
}

